applications/CompressiblePotentialFlowApplication/custom_elements/compressible_potential_flow_element.h

```diff
@@ -548,7 +571,8 @@ class CompressiblePotentialFlowElement : public Element
             if ((this)->IsDefined(ACTIVE))
                 active = (this)->Is(ACTIVE);
 
-            array_1d<double,3> v = ZeroVector();
+            array_1d<double,3> v;
+            v.clear();
```

Comment:
Better to use the constructor `array_1d (size_type array_size, value_type v)`

Reply:
I didnt find how to do this

Reply:
Try `array_1d<double,3> v(3, 0.0);`